firefox-ios/Client/Frontend/Browser/Tabs/Action/TabTrayAction.swift

```diff
@@ -31,4 +31,5 @@ enum  TabTrayActionType: ActionType {
     case didLoadTabTray
     case dismissTabTray
     case firefoxAccountChanged
+    case preserveTabs
```

Comment:
Actions should be named after the event that triggered them and not the desired outcome. In this case it should be called something related to toggling the private tab storage setting.